helm_deploy/cla-frontend/templates/_helpers.tpl

```diff
@@ -102,3 +102,14 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create the name of the service account to use
```

Comment:
Is this needed if create is always going to be false?

Reply:
Removed the create flag